adapters/33across/33across.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/mxmCherry/openrtb"
+	"github.com/mxmCherry/openrtb/v14/openrtb2"
```

Comment:
This import change makes us go from `openrtb.AnObject` to `openrtb2.AnObject` everywhere in the codebase. But, given that overtime we'll keep the OpenRTB library up to date, eventually we might get to `"github.com/.../openrtb3"` and we'll have to change every single `openrtb3.AnObject` again. Do you think we could prevent this (and simplify this PR) if we keep naming the import as `openrtb` instead?
```
  -     "github.com/mxmCherry/openrtb"
  +     openrtb "github.com/mxmCherry/openrtb/v14/openrtb2"
```

Reply:
Looks like there are some [structural differences between openrtb2 and openrtb3](https://github.com/InteractiveAdvertisingBureau/openrtb3-converter/blob/master/openrtb-3-converter-mappings.pdf), so I see both modules living together for a long time after openrtb3 is introduced.

I don't have an opinion on doing the naming change now or later, but I do think the change will be needed eventually.

Reply:
I agree with @laurb9 for renaming the package to `openrtb2` now rather than aliasing back to `openrtb`, causing us trouble in the future when we take the plunge to add OpenRTB 3 support.